docker/skr/skr-debug.sh

```diff
@@ -39,6 +39,22 @@ if [ -n "${LogLevel}" ]; then
   CmdlineArgs="${CmdlineArgs} -loglevel ${LogLevel}"
 fi
 
+if [ -n "${KeyproviderSock}" ]; then
+  CmdlineArgs="${CmdlineArgs} -keyprovider_sock ${KeyproviderSock}"
+fi
+
+if [ -n "${InFile}" ]; then
```

Comment:
These last 3 vars aren't used for the grpc server. They're used for using SKR to create "wrapped data" (ie encrypting stuff) so they wouldn't normally be used when SKR is running in ACI/AKS.